src/__tests__/core/cardanoGetAddress.spec.js

```diff
@@ -1,51 +1,373 @@
 /* @flow */
 import type { CardanoGetAddress } from '../../js/types';
+import { CARDANO_ADDRESS_TYPE } from '../../js/types';
 
 // vectors from https://github.com/trezor/trezor-firmware/tree/master/python/trezorlib/tests/device_tests/test_msg_cardano_get_address.py
 
-export const cardanoGetAddress = () => {
+const PROTOCOL_MAGICS = {
+    mainnet: 0,
```

Comment:
different values:

https://github.com/trezor/trezor-firmware/blob/c0eba979c6a1a5b6b54046c7d232743b6672291d/python/src/trezorlib/cardano.py#L22

using 0 will show "unknown network" on the device display

Reply:
Oh yes, thanks. This has also been fixed by moving the constants [here](https://github.com/trezor/connect/pull/638/commits/bd67fef49aa1297383e7500cd07f567ce1c327eb).